fvm/blueprints/contracts.go

```diff
@@ -48,6 +51,28 @@ func SetContractDeploymentAuthorizersTransaction(serviceAccount flow.Address, au
 		AddArgument(arg2), nil
 }
 
+// SetContractRemovalAuthorizersTransaction returns a transaction for updating list of authorized accounts allowed to remove contracts
+func SetContractRemovalAuthorizersTransaction(serviceAccount flow.Address, authorized []flow.Address) (*flow.TransactionBody, error) {
+	arg1, err := jsoncdc.Encode(utils.AddressSliceToCadenceValue(utils.FlowAddressSliceToCadenceAddressSlice(authorized)))
+	if err != nil {
+		return nil, err
+	}
+
+	arg2, err := jsoncdc.Encode(cadence.Path{
+		Domain:     ContractRemovalAuthorizedAddressesPathDomain,
+		Identifier: ContractRemovalAuthorizedAddressesPathIdentifier,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return flow.NewTransactionBody().
+		SetScript([]byte(setContractDeploymentAuthorizersTransactionTemplate)).
```

Comment:
Maybe rename `setContractDeploymentAuthorizersTransactionTemplate` to something that indicates it is more generally used, e.g. also for setting the removal authorizers here. Maybe `setContractOperationAuthorizersTransactionTemplate`?